libnetwork/drivers/bridge/port_mapping_linux_test.go

```diff
@@ -820,6 +821,12 @@ func TestAddPortMappings(t *testing.T) {
 				defer ul.Close()
 			}
 
+			var err error
+			d := newDriver()
+			d.nlh, err = nlwrap.NewHandle()
```

Comment:
qNot for this PR, but I noticed the weird `newDriver()` / `driver.configure()` design, which is a bit .. odd, as it looks like `driver.configure()` is the _actual_ constructor, and `newDriver()` is mostly a useless thing; (almost?) all locations where it's used, look like;

```go
d := newDriver()
if err := d.configure(options); err != nil {
 ....
}
```

We should consider making `newDriver` accept the options (if my interpretation of the flow is correct); looks like this was the commit introducing some of it; https://github.com/moby/moby/commit/d565a4df48edafa3b44230031603c24047d5ad74, although it was there in some form before that.